packages/contracts/hardhat.config.ts

```diff
@@ -45,6 +45,9 @@ const config: HardhatUserConfig = {
     version: '0.7.6',
     settings: {
       optimizer: { enabled: true, runs: 200 },
+      metadata: {
+        bytecodeHash: 'none',
```

Comment:
what does this do?

Reply:
It stops adding a hash of the bytecode at the end of the deployed bytecode for contracts. Removed under the advice of @smartcontracts as that was the agreed state before it somehow (maybe a bad rebase) was removed.